frontend/src/services/groupService.js

```diff
@@ -70,14 +23,18 @@ export const showAllGroupsService = () => {
         .catch((error) => errorHandler(error));
 };
 
+export const selectGroupService = (groupId) => {
+    store.dispatch(selectGroup(groupId));
+};
+
 export const updateGroupService = (data) => {
     return axios
         .put(GROUP_URL, data)
         .then((response) => {
-            store.dispatch(updateGroup(response.data));
+            store.dispatch(updateGroupSusses(response.data));
             selectGroupService(null);
-            getDisabledGroupsService();
-            showAllGroupsService();
+            // getDisabledGroupsService();
+            // showAllGroupsService();
```

Comment:
replace `dispatch` with `mapDispatchToProps`

Reply:
we will delete service

> replace `dispatch` with `mapDispatchToProps`

we will delete service